app/models/notification.rb

```diff
@@ -118,7 +118,7 @@ def remove_all_without_delay(notifiable_ids:, notifiable_type:)
     end
 
     def update_notifications(notifiable, action = nil)
```

Comment:
This is being called from the Article and Comment models in an `after_update` callback which means that the new models will not be committed to the database when this job runs. I believe this will be an issue as we gather up the JSON model data for the model and use it to update the notifications
```ruby
new_json_data[notifiable.class.name.downcase] = public_send("#{notifiable.class.name.downcase}_data", notifiable)
```

Because of this I believe we need to be using `after_commit` hooks to trigger this job.

Reply:
Sure @mstruve I'll make the changes

Reply:
I didn't realize there was an `after_update_commit` commit, yeah that would be the safe bet! Good call